README.md

```diff
@@ -20,40 +27,17 @@ Golang Library packages for [IndyKite Platform](https://indykite.com) Client SDK
 import "github.com/indykite/jarvis-sdk-go"
 ```
 
-In order to access to the platform you must obtain an API key first.
+In order to access to the platform you must first obtain AppAgent credentials.
 This key can be obtained either from the [Admin Console](https://console.indykite.id)
```

Comment:
"These credentials" instead of "this key", to be consistent with the change above.